target/linux/ath79/image/generic.mk

```diff
@@ -1975,6 +1975,17 @@ define Device/kuwfi_c910
 endef
 TARGET_DEVICES += kuwfi_c910
 
+define Device/kuwfi_cpe830
+  SOC := qca9533
+  DEVICE_VENDOR := KuWFi
+  DEVICE_MODEL := CPE830
+  IMAGE_SIZE := 16000k
+  IMAGES += tftp.bin
+  IMAGE/tftp.bin := $$(IMAGE/sysupgrade.bin) | yuncore-tftp-header-16m
+  DEVICE_PACKAGES := -swconfig kmod-ag71xx-legacy
```

Comment:
Why we need kmod-ag71xx-legacy?

Reply:
because this is the hardware on the board

Reply:
The driver is already compiled into the kernel:
```
target/linux/ath79/config-6.6:CONFIG_AG71XX_LEGACY=y
```